packages/create-wagmi/templates/vite-react/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>wagmi</title>
+    <script>window.global = window;</script>
+    <script type="module">
+      import process from "process";
+      import { Buffer } from "buffer";
+      import EventEmitter from "events";
+      
+      window.process = process;
+      window.Buffer = Buffer;
+      window.EventEmitter = EventEmitter;
+    </script>
```

Comment:
Pulling these out into a [polyfills.ts](https://github.com/rainbow-me/rainbowkit/blob/main/examples/with-vite/src/polyfills.ts) file could be nice so `index.html` remains (mostly) untouched.